static/js/stream_popover.js

```diff
@@ -470,6 +470,13 @@ exports.register_stream_handlers = function () {
         $(".streams_popover").on("click", "a.sp-cancel", () => {
             exports.hide_stream_popover();
         });
+        if ($(window).width() <= 768) {
+            $(".popover-inner").hide().fadeIn(300);
+            $(".popover").addClass("message-info-popover");
+        }
+        if ($(window).width() < 400) {
```

Comment:
If feels inaccurate to add this <code>if</code> statement here . Is it better to wrap it in a function such as <code>exports.color_popover_resize</code> and then call it here ?